config/routes.rb

```diff
@@ -18,5 +18,5 @@
     get 'history' => 'history#show', as: :history
   end
 
-  match 'auth/google_apps/callback' => 'callbacks#handle', via: [:get, :post]
+  get '/auth/google_oauth2/callback' => 'callbacks#handle'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.